lxc/publish.go

```diff
@@ -4,38 +4,43 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/codegangsta/cli"
```

Comment:
Insert a new line after this import to keep it separate from the github.com/lxc/lxd/* ones.